frontend/src/pages/Sessions/SessionsFeedV3/SessionQueryBuilder/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -2329,19 +2300,26 @@ function QueryBuilder(props: QueryBuilderProps) {
 				borderBottom="secondary"
 				cssClass={styles.controlBar}
 			>
-				<TimeRangeFilter
-					rule={timeRangeRule}
-					onChangeValue={(val) => updateRule(timeRangeRule, { val })}
-					onReset={
-						!readonly &&
-						timeRangeRule.val?.options[0].value !==
-							defaultTimeRangeRule.val?.options[0].value
-							? () =>
-									updateRule(timeRangeRule, {
-										val: defaultTimeRangeRule.val,
-									})
-							: undefined
-					}
+				<PreviousDateRangePicker
+					presets={presetOptions}
+					selectedDates={dateRange}
+					minDate={moment().subtract(90, 'days').toDate()}
```

Comment:
could we make this a constant outside so that it is easier to find next to the rest of the presets?